src/node/interop/interop_client.js

```diff
@@ -110,15 +110,15 @@ function emptyUnary(client, done) {
  */
 function largeUnary(client, done) {
   var arg = {
-    response_type: 'COMPRESSABLE',
-    response_size: 314159,
+    responseType: 'COMPRESSABLE',
+    responseSize: 314159,
     payload: {
       body: zeroBuffer(271828)
     }
   };
   client.unaryCall(arg, function(err, resp) {
     assert.ifError(err);
-    assert.strictEqual(resp.payload.type, 'COMPRESSABLE');
+    assert.strictEqual(resp.payload.type, 0);
```

Comment:
Now that `enumsAsStrings` is the default again, this should probably be changed back to a string.